package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "rapids.ai",
+  "name": "rapids.ai_ae",
```

Comment:
Why did this change?

Reply:
I have multiple copies of RAPIDS repo from different forks.